[2022-05-22 03:40:03,327] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Teste_ETL_Raízen.transform_n_load_dataframe manual__2022-05-22T02:35:48.168632+00:00 [queued]>
[2022-05-22 03:40:03,336] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Teste_ETL_Raízen.transform_n_load_dataframe manual__2022-05-22T02:35:48.168632+00:00 [queued]>
[2022-05-22 03:40:03,337] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-22 03:40:03,337] {taskinstance.py:1357} INFO - Starting attempt 6 of 6
[2022-05-22 03:40:03,337] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-22 03:40:03,350] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): transform_n_load_dataframe> on 2022-05-22 02:35:48.168632+00:00
[2022-05-22 03:40:03,356] {standard_task_runner.py:52} INFO - Started process 1607 to run task
[2022-05-22 03:40:03,359] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Teste_ETL_Raízen', 'transform_n_load_dataframe', 'manual__2022-05-22T02:35:48.168632+00:00', '--job-id', '82', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Raizen.py', '--cfg-path', '/tmp/tmp84jov8f4', '--error-file', '/tmp/tmpez4tii7l']
[2022-05-22 03:40:03,360] {standard_task_runner.py:80} INFO - Job 82: Subtask transform_n_load_dataframe
[2022-05-22 03:40:03,407] {task_command.py:369} INFO - Running <TaskInstance: Teste_ETL_Raízen.transform_n_load_dataframe manual__2022-05-22T02:35:48.168632+00:00 [running]> on host 541443bb6989
[2022-05-22 03:40:03,491] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Teste_ETL_Raízen
AIRFLOW_CTX_TASK_ID=transform_n_load_dataframe
AIRFLOW_CTX_EXECUTION_DATE=2022-05-22T02:35:48.168632+00:00
AIRFLOW_CTX_TRY_NUMBER=6
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-22T02:35:48.168632+00:00
[2022-05-22 03:40:21,856] {python.py:173} INFO - Done. Returned value was: None
[2022-05-22 03:40:21,870] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Teste_ETL_Raízen, task_id=transform_n_load_dataframe, execution_date=20220522T023548, start_date=20220522T034003, end_date=20220522T034021
[2022-05-22 03:40:21,898] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-22 03:40:21,925] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
